staNMF/staNMF.py

```diff
@@ -417,9 +445,18 @@ def plot(self, dataset_title="Drosophila Spatial Expression Data", xmax=0,
         Usage: Called by user to generate plot
         '''
         kArray = []
-        for i in sorted(self.instabilitydict):
-            kArray.append(i)
-            self.instabilityarray.append(self.instabilitydict[i])
+
+        if self.parallel:
+            for K in range(x1, x2+1):
+                kpath = "./staNMFDicts{}/K={}/instability.csv".format(
```

Comment:
It looks like there are many cases of hard coding paths with `/` here. I've pretty much exclusively adopted mechanisms like you have implemented [here](https://github.com/greenelab/staNMF/pull/11/files#diff-df9381e1559443f75cc2d31b7844f811R108).

I don't think you need to implement the changes in this pull request now. This could be the subject of a future PR!